src/state/application/updater.ts

```diff
@@ -20,17 +23,60 @@ function useQueryCacheInvalidator() {
   }, [chainId, dispatch])
 }
 
+function useBlockWarningTimer() {
+  const NETWORK_HEALTH_CHECK_MS = ms`15s`
```

Comment:
should these be outside of the hook so they don't get redefined each render loop?